auth-web/src/components/auth/manage-business/AffiliatedEntityList.vue

```diff
@@ -29,15 +29,73 @@
             <div class="meta">
               <v-list-item-title v-if="isNumberedIncorporationApplication(item)">Numbered Benefit Company</v-list-item-title>
               <v-list-item-title v-if="!isNumberedIncorporationApplication(item)">{{ item.name }}</v-list-item-title>
+              <v-list-item-title v-if="!item.name && isNameRequest(item.corpType.code)">{{ item.nameRequest.names[0].name }}</v-list-item-title>
```

Comment:
Does NR always sends the approved name as first one in array? If not, it cannot be assumed that the name is first one.

Reply:
In this Dashboard (old), we will just be taking the first name, the approved name.
My understanding is that these Auto Affiliated NR's wont be published until APPROVED on their side, so this should do in the meantime.

Reply:
Names publishes any state change to the NR and we add the NR to the account when the status is DRAFT, so it's not only APPROVED names but other states can also appear.
Also you may need to check the status as well, before you start a draft IA.

Reply:
Oh interesting, Andre said they were only publishing on APPROVED states. I'll look into it. 

But yes, we do look at State before enabling the ability to Create the Draft IA.

Reply:
Will update this condition, Namex is passing the individual name state 👍

Reply:
Updated to display the APPROVED name or the first name only in the event of a DRAFT.

Future state on the new dashboard WILL display all the draft/approved names, however that wasn't a requirement of the OLD dashboard.